standard/unsafe-code.md

```diff
@@ -864,6 +865,32 @@ A `char*` value produced by fixing a string instance always points to a null-ter
 > ```
 >
 > *end example*
+<!-- markdownlint-disable MD028 -->
+
+<!-- markdownlint-enable MD028 -->
+> *Example*: The following code shows a *fixed_pointer_initializer* with an expression of type other than *array_type* or `string`:
+>
+> ```csharp
+> public class C
+> {
+>     private int _value;
+>     public C(int value) => _value = value;
+>     public ref int GetPinnableReference() => ref _value;
+> }
+> public class Test
+> {
+>     unsafe private static void M()
+>     {
+>         C c = new C(10);
+>         fixed (int* p = c)
+>         {
+>             // ...
+>         }
+>     }
+> }
+> ```
+>
+> Type `C` has an accessible `GetPinnableReference` method with the correct signature. In the `fixed` statement, the `ref int` returned from that method when it is called on `c` is used to initialize the `int*` pointer `p`. *end example*
```

Comment:
Is this example actually safe though? Is it doing pinning via the magic of ref? (I can't remember the terminology.)
If this is functional but not useful, that's fine - if the example is actively dangerous (because the object could be moved) then we should call that out.

Reply:
SGTM. (It might be worth discussing whether we have or need any text in the standard that describes the impact of returning a field by reference.)

Reply:
> SGTM. (It might be worth discussing whether we have or need any text in the standard that describes the impact of returning a field by reference.)

We will. That's the subject of #213 (I'll be looking at updating it tomorrow)